### PR TITLE
Update foreman_docker to release 3.2.0 (DEB)

### PR DESCRIPTION
**Pull request by bastilian**
For plugin updates, please indicate which repos this should be built into:

* [x] Nightly
* [x] 1.15
* [ ] 1.14
* [ ] 1.13

See Foreman's [plugin maintainer documentation](http://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

File: plugins/ruby-foreman-docker/debian/changelog [hunk 1]
```diff
@@ -1,3 +1,9 @@
+ruby-foreman-docker (3.2.0-1) stable; urgency=low
+
+  * 3.2.0 released
+
+ -- Sebastian Graessl <mail@bastilian.me> Tue Jun 27 16:17:47 2017
```
Comment (mmoll) [inline plugins/ruby-foreman-docker/debian/changelog]: there needs to be one more space before the date, also the date format is wrong.

**Comment by bastilian** [comment]
@mmoll thanks for catching that! corrected the date format and spacing.

**Comment by mmoll** [comment]
merged, danke @bastilian!

**Comment by mmoll** [comment]
@bastilian @dLobatog ios this really 1.15 comptiable?
http://ci.theforeman.org/job/packaging_build_deb_plugin/1509/arch=x86,label=debian,os=jessie/console
`16:54:46 uninitialized constant ApplicationRecord`

**Pull request merged**